source/Point.h

```diff
@@ -30,15 +30,15 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 class Point {
 public:
 	Point() noexcept;
+	Point(double xy) noexcept;
```

Comment:
Where is this needed?

Reply:
Leftover from development :shrug:
May as well keep it, hell knows I've needed it before.